tests/integration/test_valid_data_in_request.py

```diff
@@ -0,0 +1,65 @@
+# -*- coding: utf-8 -*-
+# SPDX-License-Identifier: MIT
+
+import time
+
+import utils
+
+
+def test_valid_data_in_request(test_env):
+    """
+    Validate data in the request.
+
+    Process:
+    Send new request to the Cachito API
+    Send request to check status of existing request
+
+    Checks:
+    * Check that response code is 200
+    * Check in the response that state is complete
+    * Check that “packages” and “dependencies” keys have appropriate values
```

Comment:
For some reason, the quotes look off.
```suggestion
    * Check that "packages" and "dependencies" keys have appropriate values
```